trin-state/src/validation/trie.rs

```diff
@@ -394,8 +398,9 @@ mod tests {
     #[test]
     fn validate_account_state_just_leaf() {
         let address = Address::random();
-        let path: Vec<u8> = keccak(address)
-            .as_bytes()
+        let address_hash = keccak256(address);
```

Comment:
nit: Considering that `address` was created randomly and used only for `address_hash`, we could just create random `B256` (I believe `B256::random()` should work). Same below.